ember/app/ui/cases/detail/index/template.hbs

```diff
@@ -26,23 +26,13 @@
             {{t "documents.createdAt"}}
           </th>
           <td class="uk-width-1-3">
-            {{
-              if
-              this.case.createdAt
-              (moment-format this.case.createdAt "DD.MM.YYYY")
-              "-"
-            }}
+            {{format-date this.case.createdAt "dd.LL.yyyy"}}
```

Comment:
Is the `LL` format indented? And if so, why not `LLL`
e.g.: August -> Au -> Aug | `LLL` is more common.

Reply:
Thanks for the link! TIL: `LL` -> `08` & `LLL`->`Aug`